styles.css

```diff
@@ -5,15 +5,27 @@ body {
 .container {
   margin: 0 auto;
   width: 315px;
-  height: 4810px;
+  height: 5010px;
```

Comment:
- [Required] Great Job for implementing your previous reviewer suggestions 👍👌, I think you added unnecessary height here, please remove the `height: 5010px;` to stick the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👌👍
- [Optional] Kindly, you didn't add the background image, please add the background image to stick to  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)  👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/213007397-e3dbbd66-0663-43e0-8fb2-ba3cb4f81a7a.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/213007140-edcda089-df4b-4166-b830-ff3d756c37d4.png" width="200">|


😎️